src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/UntypedObject.cs

```diff
@@ -45,4 +51,94 @@ public override IEnumerable<TexlStrings.StringGetter[]> GetSignatures()
             yield return new[] { TexlStrings.IndexArg1, TexlStrings.IndexArg2 };
         }
     }
+
+    internal abstract class UntypedOrJSONConversionFunction : BuiltinFunction
```

Comment:
>UntypedOrJSONConversionFunction [](http://example.com/codeflow?start=28&length=31)

Since this is going to be used by functions outside this file (AsType / IsType), can you move it to a separate file?